make/Makefile.test

```diff
@@ -25,38 +25,41 @@ endif
 	@echo "VM started from $(ISO)"
 
 .PHONY: test-clean
-test-clean:
-	@scripts/run_vm.sh stop
+test-clean: test-stop
 	@scripts/run_vm.sh clean
 
+.PHONY: test-stop
+test-stop:
+	@scripts/run_vm.sh stop
+
 .PHONY: test-installer
 test-installer: prepare-installer-test
-	$(GINKGO) run $(GINKGO_ARGS) ./tests/installer
-	$(GINKGO) run $(GINKGO_ARGS) ./tests/smoke
+	VM_PID=$$(scripts/run_vm.sh vmpid) go run $(GINKGO) $(GINKGO_ARGS) ./tests/installer
+	VM_PID=$$(scripts/run_vm.sh vmpid) go run $(GINKGO) $(GINKGO_ARGS) ./tests/smoke
```

Comment:
Probably there are more elegant ways pass the VM pid to the test code, but so far I opted for this simple solution. This is not strictly required and can be omitted on local runs.